include/faker-cxx/Internet.h

```diff
@@ -30,6 +30,18 @@ enum class IPv4Class
     C
 };
 
+// Define the PasswordOptions struct
+struct PasswordOptions {
+    bool upperLetters;
+    bool lowerLetters;
+    bool numbers;
+    bool symbols;d
+
+    PasswordOptions(bool upper = true, bool lower = true, bool num = true, bool sym = true)
```

Comment:
please remove constructor and add default values to struct's fileds like bool upperLetters = true;

Reply:
fixed